mandatory/2-function-creation.js

```diff
@@ -16,7 +21,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  let decimalNumber = num.toFixed(2);
```

Comment:
You need to check if num has already decimal places before you round it up to two places, if you need more explaination let me know and i can help